openedx/features/course_experience/__init__.py

```diff
@@ -1,8 +1,34 @@
-# Unified course experience settings.
+"""
+Unified course experience settings and helper methods.
+"""
+
+import waffle
+
+from request_cache.middleware import RequestCache
 
 # Waffle flag to enable a single unified "Course" tab.
 UNIFIED_COURSE_EXPERIENCE_FLAG = 'unified_course_experience'
 
 # Waffle flag to enable the full screen course content view
 # along with a unified course home page.
 UNIFIED_COURSE_VIEW_FLAG = 'unified_course_view'
+
+
+def main_course_url_name(request=None):
```

Comment:
In my comment, I had mentioned using `default` instead of `main` in the name.  I get that this code is all temporary.